### PR TITLE
[Docs] Fixing php-errors in the documentation

### PR DESCRIPTION
**Pull request by mamazu**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | -none-
| License         | MIT

I am currently developing a [documentation validator](https://github.com/mamazu/documentation-parser) and added an `rst` parser to it. Sylius was the trail run project and this is what I got out of it (fixes are done manually).

One thing I am struggling with a little is the `...` situation. Having `...` in a documentation is not very helpful, because to me it is saying: "You know what to fill in here". But then why write the documentation in the first place. (eg. `SyliusOrderBundle/processors.rst`) So I am not sure about that.

### CODE REVIEW

File: docs/components_and_bundles/bundles/SyliusPromotionBundle/summary.rst [hunk 1]
```diff
@@ -31,7 +31,6 @@ Summary
                     interface: Sylius\Component\Promotion\Model\PromotionRuleInterface
                     controller: Sylius\Bundle\ResourceBundle\Controller\ResourceController
                     repository: ~
-                    factory:    Sylius\Component\Resource\Factory\Factory
```
Comment (mamazu) [inline docs/components_and_bundles/bundles/SyliusPromotionBundle/summary.rst:34]: This definition is there twice and yaml does not allow that.

File: docs/book/orders/returns.rst [hunk 2]
```diff
@@ -145,25 +145,25 @@ API coverage
...
 
-.. code-block:: php
+.. code-block:: json
 
   {
       "response": "Return request confirmed and accepted."
   }
 
-.. code-block:: php
+.. code-block:: test
```
Comment (jordisala1991) [inline docs/book/orders/returns.rst:160]: this should be text right?

Comment (mamazu) [reply docs/book/orders/returns.rst:160]: Ah, yes. This is probably why the pipeline fails. Thanks.

Comment (mamazu) [reply docs/book/orders/returns.rst]: Nope it wasn't.

File: docs/components_and_bundles/bundles/SyliusOrderBundle/processors.rst [hunk 3]
```diff
@@ -47,10 +47,10 @@ In a controller:
     <?php
 
     // Fetch order from DB
-    $order = ...;
+    $order = $this->container->get('sylius.repository.order')->find('$id');
 
     // Get the processor from the container or inject the service
-    $orderProcessor = ...;
+    $orderProcessor = $this->container->get('sylius.order_processing.shipping_charges_processor');
```
Comment (vvasiloi) [inline docs/components_and_bundles/bundles/SyliusOrderBundle/processors.rst:53]: Should be `sylius.order_processing.order_processor`

Comment (mamazu) [reply docs/components_and_bundles/bundles/SyliusOrderBundle/processors.rst:53]: Well I just took one of the processors as example.

Comment (vvasiloi) [reply docs/components_and_bundles/bundles/SyliusOrderBundle/processors.rst:53]: I thought so, but this part is about the composite order processor.

Comment (lchrusciel) [reply docs/components_and_bundles/bundles/SyliusOrderBundle/processors.rst:53]: I'm voting for `sylius.order_processing.order_processor` here

**Comment by mamazu** [comment]
Agreed. You usually want the composite one so I guess this is the best way to document it.

**Comment by lchrusciel** [comment]
@mamazu can you rebase to the master, please?

**Comment by mamazu** [comment]
Should be done now. I have also fixed on error that emerged from it. (Apparently sphinx checks if JSON data is parsable but not PHP.)

File: docs/components_and_bundles/bundles/SyliusTaxationBundle/calculating_taxes.rst [hunk 4]
```diff
@@ -36,7 +36,7 @@ Resolving rate and using calculator
 
         public function __construct(
             CalculatorInterface $calculator,
-            TaxRateResolverInterface $taxRateResolver,
+            TaxRateResolverInterface $taxRateResolver
         )
         {
```
Comment (lchrusciel) [inline docs/components_and_bundles/bundles/SyliusTaxationBundle/calculating_taxes.rst:41]:
```suggestion
        ) {
```

**Comment by lchrusciel** [comment]
Thanks, @mamazu! :tada:

**Pull request merged**